.github/workflows/ci.yml

```
@@ -22,7 +22,7 @@ jobs:
         - name: Download dependencies
           run: |
             git clone -b foxy https://github.com/eProsima/Micro-CDR src/Micro-CDR
-            git clone -b foxy https://github.com/eProsima/Micro-XRCE-DDS-Client src/Micro-XRCE-DDS-Client
+            git clone -b feature/add_history_depth https://github.com/eProsima/Micro-XRCE-DDS-Client src/Micro-XRCE-DDS-Client
```

When https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/266 merged:
```suggestion
            git clone -b foxy https://github.com/eProsima/Micro-XRCE-DDS-Client src/Micro-XRCE-DDS-Client
```